.github/ISSUE_TEMPLATE/ui.yml

```diff
@@ -11,6 +11,22 @@ body:
   - type: markdown
     attributes:
       value: Thank you for reporting!  Please also use the search to see if there are any other relevant issues or pull requests.
+  - type: markdown
```

Comment:
I don't think one makes sense. UI issues should always be in this repo.

Reply:
OK - I was on the fence with this one; in my experience people sometimes don't know what the UI really is, so it might be a problem with any "web page" they find, but we can always transfer issues if necessary.